userguide/drawing_classifier/advanced-usage.md

```diff
@@ -18,6 +18,8 @@ If you specify both `max_iterations` and `num_epochs`, `max_iterations` would
 take precedence and the model would train for that number of iterations and the 
 provided value of `num_epochs` would be ignored.
 
+When training a model for a large number of classes, it may be helpful to increase `max_iterations` to allow the model more revisions of the data to learn better.
```

Comment:
Let's phrase this advice in a way that makes it easier for the user to tell whether `max_iterations` needs to be higher. Can they look at the metrics output during training?